controllers/main.go

```diff
@@ -5,15 +5,22 @@ import (
 	"flag"
 	"os"
 
-	etcdv1 "github.com/mrajashree/etcdadm-controller/api/v1alpha3"
+	etcdv1alpha3 "github.com/mrajashree/etcdadm-controller/api/v1alpha3"
+	etcdv1 "github.com/mrajashree/etcdadm-controller/api/v1beta1"
 	"github.com/spf13/pflag"
 	"k8s.io/apimachinery/pkg/runtime"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
-	vspherev1 "sigs.k8s.io/cluster-api-provider-vsphere/api/v1alpha3"
-	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
-	controlplanev1 "sigs.k8s.io/cluster-api/controlplane/kubeadm/api/v1alpha3"
+	vspherev1alpha3 "sigs.k8s.io/cluster-api-provider-vsphere/api/v1alpha3"
+	vspherev1alpha4 "sigs.k8s.io/cluster-api-provider-vsphere/api/v1alpha4"
```

Comment:
Why do we need the `v1alpha4` ones?

Reply:
I thought we might need them for conversion webhooks, but neither v1alpha3 nor v1alpha4 are needed, so removing them